plugins/modules/accesspoint_workflow_manager.py

```diff
@@ -0,0 +1,496 @@
+from ansible.module_utils.basic import AnsibleModule
+
+import pandas as pd
+
+__metaclass__ = type
+__author__ = ("A Mohamed Rafeek, Natarajan")
+
+DOCUMENTATION = r"""
+---
+module: accesspoint_workflow_manager
+short_description: accesspoint_workflow_manager used to automate bulk AP configuration changes.
+description:
+- We can change the AP display name, AP name or Other Param based on the input.yml file
+- Using by this package we can filter specific device details like family = Switches and Hubs
+- We can compare input details with current AP configuration.
+- Desired configuration will be updated to the needed APs Only
+- Also able to reboot the Accesspoint if needed.
+version_added: '6.7.0'
+extends_documentation_fragment:
+  - accesspoint_workflow_manager
+author: A Mohamed Rafeek (@mohamedrafeek)
+        Natarajan (@natarajan)
+
+accesspoints:
+    macAddress: It is string MAC address format
+      managementIpAddress: String IP address format
+      accesspointradiotype: It is number and should be 1,2,3 and 6
+        1 - Will be 2.4 Ghz
+        2 - Will be 5 Ghz
+        3 - Will be XOR
+        6 - Will be 6 Ghz
+      apName: It should be String
+
+requirements:
+- dnacentersdk >= 2.4.5
+- python >= 3.10
+notes:
+  - dnacsdk Method used 
+"""
+
+EXAMPLES = r"""
+- name: Configure device credentials on Cisco DNA Center
+  hosts: sandboxdnac.cisco.com
+  connection: local
+  gather_facts: no
+  vars_files:
+    - "credentials.yml"
+    - "input.yml"
+  tasks:
+    - name: Get Device info and updating access point details
+      cisco.dnac.accesspoint_workflow_manager:
+        display_selection: "{{display_selection}}"
+        device_filterfield: "{{device_filterfield}}"
+        device_filter_string: "{{device_filter_string}}"
+        device_fields: "{{device_fields}}"
+        ap_selected_field: "{{ap_selected_field}}"
+        dnac_host: "{{dnac_host}}"
+        dnac_username: "{{dnac_username}}"
+        dnac_password: "{{dnac_password}}"
+        dnac_verify: "{{dnac_verify}}"
+        dnac_port: "{{dnac_port}}"
+        dnac_version: "{{dnac_version}}"
+        dnac_debug: "{{dnac_debug}}"
+        dnac_log: True
+        dnac_log_level: DEBUG
+        config_verify: True
+        dnac_api_task_timeout: 1000
+        dnac_task_poll_interval: 1
+        accesspoints: "{{ accesspoints }}"
+      register: output_list
+    - name: iterate through module output (a list)
+      debug:
+        msg: '{{ item }}'   
+        with_items: "{{output_list.output }}"
+"""
+
+RETURN = r"""
+#Case: Modification of the AP details updated and Rebooted Accesspoint
+response:
+  description: A dictionary with activation details as returned by the Catalyst Center Python SDK
+  returned: always
+  type: dict
+  sample: >
+    {
+        "response": {
+            "taskId": "string",
+            "url": "string"
+        },
+        "version": "string"
+    }
+"""
+
+class DnacAutomation(DnacBase):
```

Comment:
Can we name the class as the name of the module like Accesspoint?

Reply:
modified